notebooks/leb_salt_evaluation.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.            pipe.model.load_adapter(language)
</pre><p>Would this pipeline include a language model to improve the decoding? Wondering how that would be set, so that the right LM is loaded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/leb/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>